Yosemite/YosemiteTests/Stores/ProductVariationStoreTests.swift

```diff
@@ -290,6 +290,138 @@ final class ProductVariationStoreTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    // MARK: - ProductVariationAction.retrieveProductVariation
+
+    /// Verifies that `ProductVariationAction.retrieveProductVariation` effectively persists the retrieved product variation.
+    ///
+    func test_retrieveProductVariation_effectively_persisted() throws {
```

Comment:
Do you think this describes the scenario better? 🤔 

```suggestion
    func test_retrieveProductVariation_persists_the_ProductVariation() throws {
```

Reply:
it does read nicer! updated in https://github.com/woocommerce/woocommerce-ios/pull/2915/commits/26e5adeae232736e45eb0e54e0409e259a190c16